### ISSUE TITLE
Player from Bedrock Edition doesn't move on server-side

### ISSUE THREAD

**Issue by GladCypress3030** [GeyserMC/Geyser#884]
**Players from Bedrock Edition doesn't moving on server-side**

**To Reproduce**
1. Join the server from Java Edition and Bedrock Edition
2. Walk from Bedrock Edition
3. See from Java Edition that player from Bedrock Edition stays at one position

**Expected behavior**
Player must move on server-side

**Screenshots / Videos**

**Server Version**
1.15.2 with ViaVersion

**Geyser Version**
Geyser 1.16 as Spigot Plugin

**Minecraft: Bedrock Edition Version**
1.16

**Additional Context**

**Comment by RednedEpic** [comment]
Unable to reproduce. Please provide a video and more information if possible.

**Comment by poikNplop** [comment]
I've also experienced this on Geyser 1.16 (on Waterfall) / Waterfall+Floodgate (with a redundant ViaVersion) / Paper 1.16.1 (all on Linux/arm).

For 3 players, in order of joining the game (after switching server through Bungeecord): A (JE, Linux), B (BE, Win10), C (BE, Android); I found that player A could see neither B nor C moving (though could see the running particles); player B could see A moving but not C; player C could see everyone moving as normal.

I checked the logs and found that the only thing of note was some amount of lag (from other running programs), which may explain something, but doesn't entirely explain why some users could still see fine.

Unfortunately, I wasn't able to capture video or test further.
